frontend/src/_ui/Search/Search.jsx

```diff
@@ -95,6 +95,7 @@ export const SearchBox = ({ onChange, ...restProps }) => {
           onChange={handleChange}
           className="form-control"
           placeholder={placeholder ?? 'Search'}
+          style={{transition: "width 0.2s ease-in-out"}}
```

Comment:
Can we use a class instead of inline CSS? If we can abstract these styles into a class like `animate-width-change`, we can  reuse it for other input fields in future if we want to.